api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceV3IntegrationTests.java

```diff
@@ -77,6 +89,78 @@ void shouldAcceptAndCreateANewValidClaimWithNoNullFields() throws JsonProcessing
         verifyPostToEligibilityService();
     }
 
+    @Test
+    void shouldReturnDuplicateStatusWhenEligibleClaimAlreadyExistsForDwpHouseholdIdentifier() throws JsonProcessingException {
+        //Given
+        String householdIdentifier = "dwpHousehold1";
+        ClaimDTO dto = aValidClaimDTO();
+        Claimant claimant = aValidClaimantInSameHouseholdBuilder().build();
+        Claim claim = aValidClaimBuilder()
+                .dwpHouseholdIdentifier(householdIdentifier)
+                .claimant(claimant)
+                .build();
+        claimRepository.save(claim);
+
+        CombinedIdentityAndEligibilityResponse identityAndEligibilityResponse
+                = anIdMatchedEligibilityConfirmedUCResponseWithAllMatchesAndDwpHouseIdentifier(householdIdentifier);
+        stubEligibilityServiceWithSuccessfulResponse(identityAndEligibilityResponse);
+
+        //When
+        ResponseEntity<ClaimResultDTO> response = restTemplate.exchange(buildClaimRequestEntityForUri(dto, CLAIMANT_ENDPOINT_URI_V3), ClaimResultDTO.class);
+
+        //Then
+        assertDuplicateResponse(response);
+        verifyPostToEligibilityService();
+    }
+
+    @Test
+    void shouldReturnDuplicateStatusWhenEligibleClaimAlreadyExistsForHmrcHouseholdIdentifier() throws JsonProcessingException {
+        //Given
+        String householdIdentifier = "hmrcHousehold1";
```

Comment:
You can use the `HMRC_HOUSEHOLD_IDENTIFIER` test constant, it is exactly this value.

Reply:
Done.